LibreNMS/Device/YamlDiscovery.php

```diff
@@ -127,6 +118,51 @@ public static function discover(OS $os, $class, $yaml_data)
         return $items;
     }
 
+    /**
+     * @param Device $device Device we are working on
```

Comment:
You should fix these scrutinizer warnings

Reply:
This is now fixed